README.md

```diff
@@ -554,25 +574,27 @@ _The `vm-id` was generated by calling `subnet-cli create VMID subnetevm`. You ca
 use any value here, the only important thing is to make sure the binary you
 generate has the same name._
 
-
 #### Add New Subnet to Node Whitelist
+
 During the execution of the `wizard` command, you will be prompted to add your
 new subnetID to your node. This is done using the `whitelisted-subnets` config.
 You can provide the `whitelisted-subnets` argument by modifying your config
 file or providing an argument on startup.
 
 Example Config File:
+
 ```json
 {
-  "network-id":"fuji",
-  "health-check-frequency":"2s",
-  "log-display-level":"INFO",
-  "log-level":"INFO",
-  "whitelisted-subnets":"<PROVIDED BY SPACES-CLI>"
+  "network-id": "fuji",
+  "health-check-frequency": "2s",
+  "log-display-level": "INFO",
+  "log-level": "INFO",
+  "whitelisted-subnets": "<PROVIDED BY SPACES-CLI>"
```

Comment:
should this be SPACES-CLI?

Reply:
I think this section is under "### Run `subnet-cli wizard`" so it still shows how to use `subnet-cli`